types/react/index.d.ts

```diff
@@ -330,6 +330,10 @@ declare namespace React {
     interface FormEvent<T> extends SyntheticEvent<T> {
     }
 
+    interface InvalidEvent<T> extends SyntheticEvent<T> {
+        target: EventTarget & T;
+    }
+
```

Comment:
@vsiao oh uh I wasn't expecting this to be merged until your suggestion was applied :)  but yep this was intentional to gain the target attribute.